src/base.js

```diff
@@ -13,19 +13,27 @@ module.exports = {
     'plugin:import/recommended',
     'plugin:promise/recommended',
     'plugin:unicorn/recommended',
-    'prettier',
     'plugin:prettier/recommended',
+    'prettier',
```

Comment:
Any reason why you moved this to the end? I do not remember exactly, but the order of the imports here mattered a lot. It also could be that the order was wrong and that you fixed it, I would like to know 👍